django/contrib/admin/static/admin/js/change_form.js

```diff
@@ -12,6 +12,10 @@
                 showAddAnotherPopup(this);
             }
         });
+        // prevent double submission by disabling submit buttons
+        $('form').submit(function() {
```

Comment:
Does the admin site have any dynamically generated forms? If so, this will not work for those.